code/modules/reagents/reagents/dispenser.dm

```diff
@@ -46,6 +46,15 @@
 	reagent_state = SOLID
 	color = "#A8A8A8"
 
+/datum/reagent/metal/silver
```

Comment:
Why is this on dispenser.dm as if it were a chem dispenser reagent, yet it's not added to the list of chems on the dispenser file? Shouldn't you put it on `code/modules/reagents/reagents/other.dm` ?

Reply:
So upon further examination... liquid silver was already defined in reagents/other.dm so there wasn't any need to actually redefine it in dispenser.dm... Whoops